### PR TITLE
chore: more explicit str(Dataset) + str(empty Dataset) != ""

### PR DESCRIPTION
**Pull request by Jim-Holmstroem**

### CODE REVIEW

**Review by samedii — COMMENTED** [review]
Note that we can use slice with datasets now or we can iterate over the dataset

File: datastream/dataset.py [hunk 1]
```diff
@@ -104,10 +105,18 @@ def __len__(self):
...
+
+        content = '\n'.join(
+            list(map(
+                "    {}".format,
+                sample
+                + (['...'] if len(self) > len(sample) else [])
+            ))
+        )
+
+        return f"{self.__clas__.__name__}(\n{content}\n)"
```
Comment (FelixAbrahamsson) [inline datastream/dataset.py:119]: `self.__class__` :slightly_smiling_face:

Comment (Jim-Holmstroem) [reply datastream/dataset.py:119]: oops :P clearly I only sketched it up for the draft :)

**Comment by Jim-Holmstroem** [comment]
cool, yes, slices would be nicer :D

1) do we want to have a non-empty str() for empty datasets?
2) is it a worthy format Dataset(...) or do we want something else?

**Pull request closed without merging**